### ISSUE TITLE
Hello, where is the boundaries_sentimentint_avg.csv file ?  I can't find it. 

### ISSUE THREAD

**Issue by AotY** [A2Zadeh/TensorFusionNetwork#1]

**Comment by leocnj** [comment]
This file and several other files used by data loading have not been provided. However, as recommended by the author, data loading now should be done by using the CMU MultimodalData SDK in a much more convenient way. If using the SDK, you don't need this .csv file.

**Comment by A2Zadeh** [comment]
Hello @AotY. Please refer to the newest version of this code written in Pytorch and compatible with the CMU-Multimodal Data SDK: https://github.com/Justin1904/TensorFusionNetworks/tree/0c93a81e6c2bb9bae42d9f7bb80d54c67fbd03a8

**Comment by purvanshi** [comment]
Hello @A2Zadeh I tried the newer version of the code but there are some issues in that code. While creatin g the subnetwork for audio features the weights of first layer of MLP reach NaN.
I tried adjusting the hyperparameters (learning rate) but that doesn't help either.

**Comment by A2Zadeh** [comment]
Thanks for raising this issue. Interesting. Can you search for any NaNs in the training data? Any form of computation that reaches nan or infinity. What do you use for the training of MLP? Is this Tensor Fusion?

**Comment by purvanshi** [comment]
Hello @A2Zadeh I am using tensorfusion to train the network. 
There are some NaN values in the audio dataset due to which the whole layer weights become NaN.

**Comment by purvanshi** [comment]
Hello @A2Zadeh  where can I find the word2ix_300_mosi.pkl file?
